### ISSUE TITLE
No authentication token, authorization denied

### ISSUE THREAD

**Issue by spideyO0** [idurar/idurar-erp-crm#1049]
**Describe the bug**
No authentication token, authorization denied
**To Reproduce**
1. deploy the backend on render 
2. start the frontend by npm run dev:remote

**Expected behavior**
no authentication token cookie token not setting 

**Desktop (please complete the following information):**
 - OS: [Linux]
 - Browser [ chrome]

**Additional context**
I tried console logging the token in backend/src/controllers/middlewaresControllers/createAuthMiddleware/isValidAuthToken.js 

its giving token as undefined 

only happening when deployed backend on cloud not on localhost

**Comment by akshayneje2823** [comment]
Can I work on this

**Comment by spideyO0** [comment]
Yeah if there is any fix please let me know

**Comment by spideyO0** [comment]
@achrarajeev yeah solved but in a different way I just configured cookies as samesite: none and added withCredentials : true

**Comment by spideyO0** [comment]
Solved the issue
